Project.toml

```diff
@@ -18,9 +17,8 @@ BenchmarkTools = "1"
 DiffResults = "0.0, 1"
```

Comment:
Something I noticed in #88: Is there a reason to keep DiffResults as a proper dependency even though it is hidden with `@require`?

I think one could either remove the `@require` around `DiffResults_helpers.jl` or make it a test-only dependency (if one wants to keep the compat entries, otherwise it could be removed completely from Project.toml).

Reply:
Thanks, I will tackle that in another PR.